ui/pages/notifications/notification-components/erc721-sent-received/erc721-sent-received.tsx

```diff
@@ -112,15 +113,16 @@ export const components: NotificationComponent<ERC721Notification> = {
       type: 'body_onchain_notification',
       Image: ({ notification }) => {
         const chainId = decimalToHex(notification.chain_id);
-        const { nativeCurrencyLogo, nativeCurrencyName } =
-          getNetworkDetailsByChainId(`0x${chainId}` as keyof typeof CHAIN_IDS);
         return (
           <NotificationDetailNft
-            networkSrc={nativeCurrencyLogo}
-            tokenId={notification.data.nft.token_id}
-            tokenName={notification.data.nft.name}
-            tokenSrc={notification.data.nft.image}
-            networkName={nativeCurrencyName}
+            nft={{
+              address: notification.data.nft?.collection.address || '',
+              chainId,
+              image: notification.data.nft?.image || '',
+              name: notification.data.nft?.name || '',
+              standard: TokenStandard.ERC1155,
```

Comment:
Shouldn't we se the standard for this file to ERC721?

If this is blocking the approvals, can we do this in a separate PR?

Reply:
In fact, why is the `standard` property added? I checked `nft-item.js` and I don't think it is used. It was a little hard to verify since the file is in JS